pyspatialopt/models/binary_mclp_distance_matrix.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: UTF-8 -*-
+import logging
+import sys
```

Comment:
I pulled down your branch and the `sys` and `arcpy` modules are not used, can you remove them?